EIPS/eip-7496.md

```diff
@@ -51,6 +52,8 @@ The `traitKey` is used to identify a trait. The `traitKey` MUST be a unique `byt
 
 The `traitKey` SHOULD be a `keccak256` hash of a human readable trait name.
 
+If a `traitKey` is queried that has not been set, the contract MUST revert with `error TraitKeyNotSet(bytes32 traitKey)` to identify that the trait value should not be used.
```

Comment:
What does this requirement buy us? We allow the value to be unset in the spec, so reverting shouldn't be strictly necessary

Reply:
I'm thinking of "unset" to mean "does not have a value set for it yet, but is a valid key". I think this is more a case of "your trait key is invalid".

In either case though, I don't think reverting is strictly necessary. If I wanted to add a new trait to my token, I could just use the default value and update the metadata URI without making any updates onchain. Confused on what the use case would be to require a revert here vs allowing either a revert or the default value to be returned.

Reply:
ok will remove